test/fprime_gds/common/gds_cli/filtering_utils_test.py

```diff
@@ -54,7 +54,6 @@ def sample_event(sample_template):
         filtering_utils.get_search_predicate("the same combination"),
     ],
 )
-@pytest.mark.gds_cli
```

Comment:
Do we need to remove these markers to get everything working? I haven't used pytest much, so I definitely don't know what I'm talking about, but I assumed that pytest would just run everything by default. If tests have already been marked they're probably worth keeping just on the off chance somebody running locally just wants to run the cli tests or something.

Reply:
Yeah, if you have the markers but don't define them then it gives a bunch of errors. Should I add them back in then?

Reply:
Ah, I get it now. Yeah, add the pytest.ini file back in. Something tells me nobody has been running these unit tests anytime recently.